mne/viz/tests/test_3d.py

```diff
@@ -637,4 +641,27 @@ def test_brain_colorbar(orientation, diverging, lims):
     plt.close('all')
 
 
+@requires_pysurfer
+@testing.requires_testing_data
+@traits_test
+def test_mixed_sources_plot_surface():
+    """Test plot_surface() for  mixed source space."""
+    src = read_source_spaces(fwd_fname2)
+    N = np.sum([s['nuse'] for s in src])  # number of sources
+
+    T = 2  # number of time points
+    S = 3  # number of source spaces
+
+    data = rng.randn(N, T)
+    vertno = S * [np.arange(N // S)]
+
+    stc = MixedSourceEstimate(data, vertno, 0, 1)
+
+    # This requires FreeSurfer to be installed, so that the subject
+    # fsaverage exists.
```

Comment:
this is not really true. You can have fsaverage in yoru subjects dir without install freesurfer. Just remove this line.